content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -220,11 +220,53 @@ After 15 seconds, view Pod events to verify that liveness probes:
 kubectl describe pod goproxy
 ```
 
+## Define a gRPC liveness probe
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+If your application implements [gRPC Health Checking Protocol](https://github.com/grpc/grpc/blob/master/doc/health-checking.md),
+kubelet can be configured to use it for application liveness checks.
+
+{{< codenew file="pods/probe/content/en/examples/pods/probe/grpc-liveness.yaml">}}
+
+For the gRPC probe `port` have to be configured. If the health endpoint is configured
+on non-default service, `service` must be configured.
+
+{{< note >}}
+Unlike http and tcp probes, named ports cannot be used and custom host cannot be configured.
+{{< /note >}}
+
+Configuration problems - incorrect port and service, or not implemented health
```

Comment:
suggestion:  "Configuration problems (e.g. incorrect port and service, unimplemented health checking protocol) are considered a probe failure, similar to HTTP and TCP probes."